packages/aws-cdk-lib/aws-stepfunctions/lib/state-graph.ts

```diff
@@ -35,6 +35,16 @@ export class StateGraph {
    */
   public timeout?: Duration;
 
+  /**
+   * Set a comment to render into the graph JSON.
+   *
+   * Read/write. Only makes sense on the top-level graph, subgraphs
+   * do not support this feature.
```

Comment:
I understand you followed the same pattern as the `timeout` property - but this behavior makes me a little uncomfortable. Lets see if we can make it better?

See other comments.